Windows build is broken with VS 15.9.4

Hello,
I believe this is the same problem as https://github.com/servo/servo/issues/22476 but happening locally. It started after updating my Visual Studio to the latest version 15.9.4.

> --- stderr
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\yvals_core.h:294:5: error: STL1000: Unexpected compiler version, expected Clang 7 or newer.
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\yvals_core.h:294:5: error: STL1000: Unexpected compiler version, expected Clang 7 or newer., err: true

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,5 +1,5 @@
 version: 1.0.{build}
-image: Previous Visual Studio 2017
+image: Visual Studio 2017
 
 environment:
   CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"

--- a/python/servo/packages.py
+++ b/python/servo/packages.py
@@ -4,7 +4,7 @@
 
 WINDOWS_MSVC = {
     "cmake": "3.7.2",
-    "llvm": "6.0.0",
+    "llvm": "7.0.0",
     "moztools": "3.2",
     "ninja": "1.7.1",
     "openssl": "1.1.0e-vs2015",

